client/client-multi/client-c/toolchains/esp8266.cmake

```diff
@@ -0,0 +1,20 @@
+include(CMakeForceCompiler)
+
+#change this to path where you've installed xtensa toolchain
+set(ESP_TOOLCHAIN_BASE /opt/Espressif/crosstool-NG/builds/xtensa-lx106-elf/bin)
```

Comment:
I think it's better to add one more variable, something like ESPRESSIF_HOME. The user can specify it either as a system variable or through CMake "-DVAR_NAME"  syntax. If we will do this, documentation need to be updated too.